dynatrace-aws-log-forwarder-template.yaml

```diff
@@ -578,6 +578,9 @@ Resources:
             DeleteOnTermination: "true"
             VolumeSize: "8"
       SecurityGroupIds: [ !Ref VPCSecurityGroup ]
+      Tags:
+        - Key: Name
+          Value: !Join [ "-", [ !Ref 'AWS::StackName', "active-gate" ] ]
```

Comment:
Funny that name's a tag. I assume it works:P

Reply:
![image](https://user-images.githubusercontent.com/32064437/126648242-026f781e-2459-4cbd-988c-1713c16adcc9.png)

Reply:
It is kind of weird how it's a tag but from what I can gather that's how it is and it does work.

Reply:
Can confirm, it is a tag